nio/client/async_client.py

```diff
@@ -1241,6 +1243,96 @@ def add_response_callback(
             if self.olm.inbound_group_store.add(session):
                 self.store.save_inbound_group_session(session)
 
+    @store_loaded
```

Comment:
Is that a tab completion error? We need to be logged in but do not need the store to be loaded.